Jenkinsfile

```diff
@@ -0,0 +1,94 @@
+                                                   replaceTokens: true,
+                                                   targetLocation: 'gradle.properties')]) {
+                        withCredentials([usernamePassword(credentialsId: 'dse-artifactory',
+                                         usernameVariable: 'ARTIFACTORY_USER',
+                                         passwordVariable: 'ARTIFACTORY_PASSWORD')]) {
+                            withAnt(installation: 'ant-1.10.7') {
+                                sh "curl -O https://repo1.maven.org/maven2/org/apache/ivy/ivy/2.5.2/ivy-2.5.2.jar"
+                                script {
+                                    if (!fileExists('/home/ubuntu/.ant/')) {
+                                        sh "mkdir /home/ubuntu/.ant/"
+                                    }
+                                    if (!fileExists('/home/ubuntu/.ant/lib')) {
+                                        sh "mkdir /home/ubuntu/.ant/lib"
+                                    }
+                                }
+                                sh "cp ivy-2.5.2.jar /home/ubuntu/.ant/lib/"
+                                sh "export ARTIFACTORY_USER=$ARTIFACTORY_USER"
+                                sh "export ARTIFACTORY_PASSWORD=$ARTIFACTORY_PASSWORD"
+                                sh "ant publish"
+                                // sh "ant installer"
+                                sh "mkdir artifacts"
+                                sh "cp dist/jython-standalone* artifacts/"
+                            }
+                        }
+                    }
+                archiveArtifacts artifacts: 'dist/jython-standalone-*.jar', onlyIfSuccessful: true, defaultExcludes: false, caseSensitive: false
+                archiveArtifacts artifacts: 'dist/jython-installer.jar', onlyIfSuccessful: true, defaultExcludes: false, caseSensitive: false
+            }
+        }
+        stage ('Upload to artifactory') {
+            steps {
+                sh "ls artifacts/ > commandResult"
+
+                script {
+                    env.jarfilename = readFile('commandResult').trim()
+                }
+                sh "echo $jarfilename"
+
+                withCredentials([usernamePassword(credentialsId: 'dse-artifactory',
+                                                  usernameVariable: 'ARTIFACTORY_USER',
+                                                  passwordVariable: 'ARTIFACTORY_PASSWORD')]) {
+                    // sh "curl -sSf -u '$ARTIFACTORY_USER:$ARTIFACTORY_PASSWORD' -X PUT -T artifacts/$jarfilename 'https://repo.aws.dsinternal.org/ui/native/dse/com/datastax/opscenter/jython-standalone/2.7.3a1/$jarfilename'"
+                    sh "curl -v --user $ARTIFACTORY_USER:$ARTIFACTORY_PASSWORD --data-binary artifacts/$jarfilename -X PUT 'https://repo.aws.dsinternal.org/ui/native/dse/com/datastax/opscenter/jython-standalone/2.7.3a1/$jarfilename'"
```

Comment:
I've been poking at this problem again today and after going through the ui to manually upload the artifact I'm seeing something that might be the problem. The repo the jython artifact is in is 'datastax-public-releases-local'. Any attempt to upload the artifact you make should include that somewhere. In the case here I think it either replaces the 'dse' portion of the url is comes after it.

Here is a link to the artifactory page showing the test artifact I was able to upload and the reference from the opscenterd dependencies: 
https://repo.aws.dsinternal.org/ui/repos/tree/General/datastax-public-releases-local/com/datastax/opscenter/jython-standalone/2.7.3a1-test

I hope to be able to test the upload using curl, but I'm not sure if I'll have time during what is left of my workday.

Reply:
The upload is working for with curl now with this change @orion104, Thank you!

Does this mean we won't need to use ivy to upload to the artifactory anymore? If so, I will cleanup the ivy related changes in the code and have only the relevant changes.